cli/command/service/create.go

```diff
@@ -74,6 +74,8 @@ func runCreate(dockerCli *command.DockerCli, opts *serviceOptions) error {
 
 	ctx := context.Background()
 
+	resolveServiceImageDigest(dockerCli, &service)
```

Comment:
Should service create fail if the image fails to resolve the reference against DCT?